user_docs/en/userGuide.t2t

```diff
@@ -1585,9 +1585,11 @@ The selection indicator is not affected by this option, it is always dots 7 and
 ==== Show Messages ====[BrailleSettingsShowMessages]
 This is a combobox that allows you to select if NVDA should display braille messages and when they should disappear automatically.
 
+To toggle show messages from anywhere, please assign a custom gesture using the [Input Gestures dialog #InputGestures].

```

Comment:
Open question for NVAccess reviewer:
This type of comment is added for some scripts, and sometimes not. What is the opinion of NVAccess on this topic?
* Should it be added for each GUI control supporting an associated script?
* Should this comment be factorized for all settings and put at the beginning of the settings description?
* Should it be just removed from all settings paragraphs if we have checked that the description of the Input dialog is enough?
* Or should we continue as today, letting the PR's authors decide if it should be added or not?

Cc @Qchristensen

Reply:
I think if you can do something with / to / in NVDA, it should be documented somewhere.  I think having it documented where the main description of that feature is makes most sence (I considered creating a table of all the unassigned scripts - and while that might be helpful for someone wanting to learn "what else NVDA can do" that doesn't have keystroke - the chance is it will quickly become out of date as authors won't remember to document there) - I have an intention of going throuhg all the unassigned scripts and ensuring they are all documented, so a line like this for now is good.